source/__tests__/integration/profile-create.test.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path')
+
+describe('Sinopia Profile Editor imports a Profile', () => {
+
+  beforeAll(async () => {
+    await page.goto('http://127.0.0.1:8000/#/profile/create')
+  })
+
+  describe('Import Data Modal Upload Dialog', () => {
+
+    beforeAll(async () => {
+      await expect(page).toClick('a', { text: 'Import'})
+    })
+
+    it('click the import button and display dialog', async () => {
+      await expect(page).toMatch(/Import Data/)
+    })
+
+
+    it('uploads a local BIBFRAME 2.0 Item Profile and checks for loaded Profile', async () => {
+      const bf_item_location = path.join(__dirname, "..", "__fixtures__", 'item.json')
+
+      await expect(page).toUploadFile(
+        'input[type="file"]',
+        bf_item_location,
+      )
+
+      await page.waitFor(1000) // Waits for the Profile to load
+      await expect(page).toMatch(/BIBFRAME 2.0 Item/)
```

Comment:
I like these matchers as they capture what we'd look for specific to the Item Profile